sdk/patina/src/uefi_protocol/performance_measurement.rs

```diff
@@ -40,7 +40,7 @@ pub enum PerfAttribute {
 }
 
 /// Function to create performance record with event description and a timestamp.
-pub type CreateMeasurement = unsafe extern "efiapi" fn(
+pub type CreateMeasurementExt = unsafe extern "efiapi" fn(
```

Comment:
I'm not sure `Ext` is the most accurate name here. Its not an extension of core functionality, it is the core functionality. I would consider renaming this. I honestly can't think of a much better name. Maybe `Uefi` instead. Interested to know if others have an opinion.